Gopkg.lock

```diff
@@ -189,10 +195,16 @@
 
 [[projects]]
   name = "github.com/shirou/gopsutil"
-  packages = ["host","internal/common","mem","process"]
+  packages = ["cpu","host","internal/common","mem","net","process"]
   revision = "bfe3c2e8f406bf352bc8df81f98c752224867349"
   version = "v2.17.11"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/shirou/w32"
```

Comment:
it doesn't appear that this windows wrapper is necessary for xxhash. is there a reason you included that here?

Reply:
no clue where that came from, I'll remove